src/observers/KmlObserverBase.hpp

```diff
@@ -285,7 +285,7 @@ class KmlObserverBase : public openfluid::ware::PluggableObserver
         Args << "a";
         Args << "-tzip";
         Args << QString::fromStdString(KmzFilePath);
-        Args << QString::fromStdString(InputDir);
+        Args << QString::fromStdString(InputDir)+"/*";
```

Comment:
Check if syntax works on Windows systems

Reply:
Works on windows, other arguments (KmzFilePath/InputDir) also containing slashes. Need general cleanup of paths?

Reply:
No need to cleanup, slashes works for both Unices/Windows systems. Backslashes ares used only for displaying paths in UI for better consistency in UX